package.json

```diff
@@ -31,14 +31,20 @@
   "dependencies": {
     "@zenginehq/post-rpc-client": "^0.8.1",
     "ace-builds": "1.4.6",
+    "angular": "1.2.27",
     "angular-bootstrap": "github:Wizehive/bootstrap-bower",
+    "angular-local-storage": "0.1.4",
+    "angular-sanitize": "1.2.27",
     "angular-ui-ace": "github:Wizehive/ui-ace",
+    "angular-ui-sortable": "0.13.4",
```

Comment:
@wizehive-jwatt I know we originally agreed to use the wizehive fork, but knowing that it's much harder to unpublish an npm module after the left-pad issue, are there other concerns about just getting this directly from the third party?